USAGE_android.md

```diff
@@ -361,6 +361,50 @@ in the GFXReconstruct GitHub repository at `layer/vk_layer_settings.txt`. Most
 binary distributions of the GFXReconstruct software will also include a sample
 settings file.
 
+#### Layer Settings via VkLayerSettingEXT
```

Comment:
I think we should refer to this mechanism by the extension name `VK_EXT_layer_settings`